.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "tools/OpenROAD"]
 	path = tools/OpenROAD
 	url = ../OpenROAD.git
+[submodule "tools/LSOracle-Plugin"]
+	path = tools/LSOracle-Plugin
+	url = https://github.com/The-OpenROAD-Project/LSOracle-Plugin.git
```

Comment:
LSOracle-Plugin.git has yosys as a submodule which means we'll wind up with two (possibly different) copies of yosys.  We need to avoid that.

Reply:
There is no requirement that this copy be checked out as far as using the build - it can use the other copy. As long as there is yosys headers to link against in the YOSYS_DIR location passed to the makefile, it will work.

Reply:
We check out with --recursive and it would be an extra problem to avoid this one case.  Also you are potentially building against a different version of yosys than you will run against.  Is that an issue?

Reply:
In principle it could be a problem - but there's no linking done against any source in yosys. As long as there isn't some catastrophic change in the yosys header files for plugins, it would be compatible.

Reply:
If you don't need to yosys source then why is it a submodule?